### ISSUE TITLE
Scroll wheel support when choosing time

### ISSUE THREAD

**Issue by sich97** [nextcloud/nextcloud-vue#824]
Version 2.0 now makes it so you choose time of day by using a vertical slider to scroll though both hours and minutes. However, I'm not able to use the scroll wheel on my mouse to do this. Support for this should be implemented.
![Screenshot_20200118_024616](https://user-images.githubusercontent.com/7852398/72656608-c457ef00-399c-11ea-83fa-0e7537843508.png)

**Comment by dartcafe** [comment]
Notice: On my side scrolling works in Chrome and Firefox (Windows).

**Comment by sich97** [comment]
> Notice: On my side scrolling works in Chrome and Firefox (Windows).

I'm using Firefox on Ubuntu 19.10 (Pop_OS).
I have experienced no issues with scrolling elsewhere on my system.

**Comment by susnux** [comment]
I can not reproduce this with current version 7.12 or master using Firefox 113 on Linux and Windows.
Probably this was already fixed.
